src/main/java/mate/academy/intro/repository/BookRepositoryImpl.java

```diff
@@ -43,4 +44,11 @@ public List<Book> findAll() {
             throw new RuntimeException("Can't find all books from DB", e);

```

Comment:
```suggestion
            throw new DataProcessingException("Can't find all books from DB", e);
```

for each exception on Repo layer